internal/controllers/core/portforward/reconciler_test.go

```diff
@@ -32,14 +37,21 @@ func TestCreatePortForward(t *testing.T) {
 	f := newPFRFixture(t)
 	defer f.TearDown()
 
-	f.onChange()
+	f.wait()
 	require.Equal(t, 0, len(f.r.activeForwards))
 
-	state := f.st.LockMutableStateForTesting()
-	state.PortForwards[pfFooName] = f.makeSimplePF(pfFooName, 8000, 8080)
-	f.st.UnlockMutableState()
-
-	f.onChange()
+	pf := f.makeSimplePF(pfFooName, 8000, 8080)
+	f.Create(pf)
+
+	// gonna want to test that
+	//   a. the PFs are started as expected (r.activeForwards etc.) and
+	//   b. the status is reflected in the API
+	time.Sleep(time.Second)
+	f.requireState(pfFooName, func(pf *PortForward) bool {
```

Comment:
I think the `time.Sleep()` can go away?

Also, this needs a real assertion 😬

Reply:
lol that was supposed to go away 😅